bin/convert-argv.js

```diff
@@ -537,7 +537,7 @@ module.exports = function(yargs, argv, convertOptions) {
 				if(i < 0 || (j >= 0 && j < i)) {
 					var resolved = path.resolve(content);
 					if(fs.existsSync(resolved)) {
-						addTo("main", resolved);
+						addTo("main", `${resolved}${path.extname(resolved) ? "" : path.sep}`);
```

Comment:
Please check if it's a folder with `fs.statsSync` `isDirectory`. Try to replace the `fs.existsSync` call, it's deprecated anyway.